packages/kotti-table/src/TableRow.js

```diff
@@ -23,19 +23,22 @@ export default {
 			handleClick,
 			handleFocus,
 			handleBlur,
+			handleKey,
 			handleExpand,
+			handleActionsClick,
 			hasActions,
 			renderActions,
 			isDisabled,
 		} = this
 		return (
 			<tr
 				class={_trClasses}
-				role={isInteractive ? 'button' : false}
-				tabindex={isInteractive ? 0 : false}
+				role={isInteractive && !isDisabled ? 'button' : false}
```

Comment:
Why would you tab to the row if the row is disabled? Why would it act as a button if it’s disabled?

Reply:
you shouldn't be able to tab to the row if it's disabled, so if disabled is true you won't be able too

not isDisabled means that if isDisabled is true then tabindex is false 

The button one may be unnecessary to switch from being evaluated as a button to not, but I figured not to confuse screen readers